sdk/go/internal/engineconn/bin/bin.go

```diff
@@ -0,0 +1,54 @@
+package embedded
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+
+	"dagger.io/dagger/internal/engineconn"
+	"github.com/docker/cli/cli/connhelper/commandconn"
+)
+
+func init() {
+	engineconn.Register("bin", New)
```

Comment:
Why `bin`?
Why not `stdio`? (is that what would be the defined scheme? (xxx://)